docker/dev/docker-compose.yml

```diff
@@ -1,6 +1,16 @@
 version: '3.3'
 
 services:
+  nginx:
+    image: nginx:1-alpine
+    ports:
+      - "1236:1236"
```

Comment:
I wanted to use port 1234 here for consistency, but that wasnt working because image uploads were sent to localhost:8536 in that case, so the jwt cookie wasnt included. This is because of this overly complicated code in lemmy-ui, which seems somewhat broken.

https://github.com/LemmyNet/lemmy-ui/blob/main/src/shared/env.ts